js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -44,6 +45,7 @@ export function useAssetGraphData(opsQuery: string, options: AssetGraphFetchScop
       groupSelector: options.groupSelector,
     },
   });
+  useQueryResultDependency(fetchResult, 'ASSET_GRAPH_QUERY');
```

Comment:
I suspect this will be the primary way developers will interact with traces. Explicitly marking queries as display done dependencies this way.

Reply:
i guess the import path scopes it to tracing, but seems like there could be a name that was more intuitive on first glance 

`useBlockTraceOnQueryResult` 
`useTraceQueryResult` 
`useTraceDependsOnQueryResult`
`useTraceChildQueryResult`
`useAddQueryToTrace`

we don't need to get this right out the gate since rename is easy, but worth putting som thought in to if its the primary API that will spread across the code base